library/build.gradle

```diff
@@ -37,6 +37,10 @@ android {
     lintOptions {
         abortOnError false
     }
+
+    testOptions {
+        unitTests.returnDefaultValues = true
```

Comment:
Otherwise the test that executes

```
ContentValues values = new ContentValues();
values.put(Downloads.Impl.COLUMN_STATUS, mStatus);
```

will fail